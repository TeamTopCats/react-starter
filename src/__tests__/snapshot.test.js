// // Link.react.test.js
// import React from 'react';
// import Link from '../Link.react';
// import renderer from 'react-test-renderer';

test('test', () => {})

// test('Link changes the class when hovered', () => {
//   const component = renderer.create(
//     <Link page="http://www.facebook.com">Facebook</Link>,
//   );
//   let tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   tree.props.onMouseEnter();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   tree.props.onMouseLeave();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
// });

// // __tests__/__snapshots__/Link.react.test.js.snap
// exports[`Link changes the class when hovered 1`] = `
// <a
//   className="normal"
//   href="http://www.facebook.com"
//   onMouseEnter={[Function]}
//   onMouseLeave={[Function]}>
//   Facebook
// </a>
// `;

// exports[`Link changes the class when hovered 2`] = `
// <a
//   className="hovered"
//   href="http://www.facebook.com"
//   onMouseEnter={[Function]}
//   onMouseLeave={[Function]}>
//   Facebook
// </a>
// `;

// exports[`Link changes the class when hovered 3`] = `
// <a
//   className="normal"
//   href="http://www.facebook.com"
//   onMouseEnter={[Function]}
//   onMouseLeave={[Function]}>
//   Facebook
// </a>
// `;
