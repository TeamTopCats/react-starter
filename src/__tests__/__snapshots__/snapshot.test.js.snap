// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`отображается корректно 1`] = `<a />`;
